src/Feed.js

```diff
@@ -125,7 +125,7 @@ class Feed {
       })
       .on('response', (res) => {
         if (res.statusCode !== RESPONSE_CODES.OK) {
-          this.handleError(new FeedError(`This URL returned a ${res.statusCode} status code`, 'fetch_url_error', this.url));
+          this.handleError(new FeedError(`${this.url} returned a ${res.statusCode} status code`, 'fetch_url_error', this.url));
```

Comment:
```suggestion
          this.handleError(new FeedError(`This URL returned a ${res.statusCode} status code`, 'fetch_url_error', this.url));
```

Keep the old format here and add custom formatting in the FeedError#toString